models/event.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var EventSchema = new mongoose.Schema({
```

Comment:
What do you think about using the OSDI event as the schema for persistence?

https://opensupporter.github.io/osdi-docs/events.html